source/tasks/BuildInformation/BuildInformationV5/buildInformation.ts

```diff
@@ -93,7 +93,7 @@ export class BuildInformation {
             this.tool.arg(["--package-id", item]);
         }
 
-        await executeTask(this.tool, this.connection, "Build information successfully pushed.", "Failed to push build information.", additionalArguments);
+        await executeTask(this.tool, "(build-information:push:v5)", this.connection, "Build information successfully pushed.", "Failed to push build information.", additionalArguments);
```

Comment:
The colon should be a semi-colon. This aligns with the cli, plus aligns with the other sub-context data in other user-agent strings